.golangci.yml

```diff
@@ -237,7 +237,7 @@ linters:
     gocognit:
       # Minimal code complexity to report.
       # Default: 30 (but we recommend 10-20)
-      min-complexity: 20
+      min-complexity: 30
```

Comment:
linter complains high complexity within `matchDeps` after several round of simplification.  I'll leave the final decision to reviewers. If reviewers agree that 20 is a reasonable value, I will simplify matchDeps further